overlord/devicestate/devicemgr.go

```diff
@@ -248,6 +248,8 @@ func (m *DeviceManager) ensureOperational() error {
 	}
 
 	// if there's a gadget specified wait for it
```

Comment:
the comment seems wrongly placed now, or the spaces a bit wrong.

maybe

     var gadgetInfo ... 
     var  hasPrepareDeviceHook ...
     // comment
     if gadget != ""